pandas/io/spss.py

```diff
@@ -0,0 +1,37 @@
+from pathlib import Path
+from typing import Optional, Sequence, Union
+
+from pandas.core.api import DataFrame
+
+
+def read_spss(path: Union[str, Path],
+              usecols: Optional[Union[str, Sequence[str]]] = None,
```

Comment:
I don't think this is correct, usecols (in all of our other parsers) must be a list-like or None.

can you change this (and add a test for non-list-like)

Reply:
@bashtage we have the columns arg on read_stata, this is simliar, yes? (maybe should change read_stata to accept usecols rather than columns as its more consistent with the other IO readers)

Reply:
`pyreadstat` requires `usecols` to be a `list` (`tuple` or other list-likes raise a `TypeError`). Do you want me to just re-raise this exception, or should I try to convert list-likes to a `list` and raise a custom exception if this is not possible?

Reply:
Ideally we would accept any sequence and convert it to a list for pyreadstat.

Reply:
Done, `usecols = list(usecols)` should give a clear error message if `usecols` is not a sequence.